task/icr-scan/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- openrefactory
```

Comment:
you might have to join the tektoncd org in order to author this task? btw we have started migrating from centralized repo to decentralized one, eg https://github.com/tektoncd-catalog/git-clone. Will it be possible for you to have this in your own account?

Reply:
@vinamra28 , after reading the contribution guideline, I have reached out to Vincent Deemester via email to be added as a read-only collaborator in the tektoncd org. Is it the correct approach?  I will be glad if you can shed some light.

Reply:
hey @openrefactory, can you please raise a PR in tektoncd/community to get added yourself into the community so that you can maintain this task ? example: https://github.com/tektoncd/community/pull/869

Reply:
@vinamra28 Thanks a lot. I have created a PR (https://github.com/tektoncd/community/pull/1021)
Can you please suggest what needs to be done so that the 'failed' tests are resolved?